### PR TITLE
feat code snippet

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the code snippet and fixed some of the typos 
No major changes.
Screenshot (478)

### CODE REVIEW

File: Doc/avatar.html [hunk 1]
```diff
@@ -58,29 +60,66 @@ <h3>Components</h3>
...
-                <img class="avatar avatar-square avatar-xs" src="../Assets/images/sample-img.png" alt="" srcset="">
-                <img class="avatar avatar-square avatar-sm" src="../Assets/images/sample-img.png" alt="" srcset="">
-                <img class="avatar avatar-square" src="../Assets/images/sample-img.png" alt="" srcset="">
-                <img class="avatar avatar-square avatar-lg" src="../Assets/images/sample-img.png" alt="" srcset="">
-                <img class="avatar avatar-square avatar-xl" src="../Assets/images/sample-img.png" alt="" srcset="">
+                <img class="avatar avatar-square avatar-xs" src="../Assets/images/sample-img.png">
+                <img class="avatar avatar-square avatar-sm" src="../Assets/images/sample-img.png">
+                <img class="avatar avatar-square" src="../Assets/images/sample-img.png">
+                <img class="avatar avatar-square avatar-lg" src="../Assets/images/sample-img.png">
+                <img class="avatar avatar-square avatar-xl" src="../Assets/images/sample-img.png">
```
Comment (tanishq20) [inline Doc/avatar.html]: `alt` must be there if `img` tag is used.

File: Doc/badge.html [hunk 2]
```diff
@@ -97,6 +131,21 @@ <h2 class="margin-top-bottom">Avatar Badge Outline</h2>
...
+                    <code class="html">
+        <div class="component-display-div-row maple-flex-xy-center">
+            <div class="badge">
+                <img class="avatar dnd-border" src="../Assets/images/sample-img.png" >
+            </div>
+            <div class="badge">
+                <img class="avatar offline-border" src="../Assets/images/sample-img.png" >
+            </div>
+            <div class="badge">
+                <img class="avatar online-border" src="../Assets/images/sample-img.png" >
```
Comment (tanishq20) [inline Doc/badge.html]: Here also add `alt` in `img` tag.

File: Doc/card.html [hunk 3]
```diff
@@ -86,6 +88,37 @@ <h4>by Fujiko Fujio</h4>
                         </div>
                     </div>
                 </div>
+                <pre>
+                    <code class="html">
+        <div class="light-background component-display-div-row maple-flex-xy-center ">
+            <div class="card card-relative">
+                <div class="header">
+                    <div class="header-image-text-div">
+                            <img class="header-image" src="../Assets/images/dore.jpg">
```
Comment (tanishq20) [inline Doc/card.html]: missing `alt`

File: Doc/card.html [hunk 4]
```diff
@@ -114,13 +147,48 @@ <h4>by Fujiko Fujio</h4>
...
                 </div>
+                <pre>
+                    <code class="html">
+            <div class="light-background component-display-div-row maple-flex-xy-center">
+                <div class="card card-relative card-dismiss">
+                    <div class="header">
+                        <div class="header-image-text-div">
+                                <img class="header-image" src="../Assets/images/dore.jpg">
+                        </div>   
+                        <div class=" pd-horizontal">
```
Comment (tanishq20) [inline Doc/card.html]: remove gap in class name

File: Doc/card.html [hunk 5]
```diff
@@ -114,13 +147,48 @@ <h4>by Fujiko Fujio</h4>
...
+                </div>
+            </div>
+                    </code>
+                </pre>
                 <h2 class="margin-top-bottom">Text overlay card</h2>
                 <div class="component-display-div-row maple-flex-xy-center light-background">
                     <div class="card ">
                         <div class="header card-relative">
                             
                                     <img class="header-image" src="../Assets/images/dore.jpg" alt="" srcset="">
```
Comment (tanishq20) [inline Doc/card.html]: `alt` and `srcset` should not be empty.

File: Doc/card.html [hunk 6]
```diff
@@ -190,21 +312,47 @@ <h4>by Fujiko Fujio</h4>
                         </div>
                     </div>
                 </div>
+                <pre>
+                    <code class="html">
+            <div class="component-display-div-row maple-flex-xy-center light-background">
+                <div class="card">
+                    <div class="header maple-flex">
+                        <div class="header-image-text-div">
+                                <img class="header-image" src="../Assets/images/dore.jpg">
```
Comment (tanishq20) [inline Doc/card.html]: `alt` missing.

File: Doc/card.html [hunk 7]
```diff
@@ -190,21 +312,47 @@ <h4>by Fujiko Fujio</h4>
...
+                </div>
+            </div>
+                    </code>
+                </pre>
                 <h2 class="margin-top-bottom">Shadow card</h2>
                 <div class="component-display-div-row maple-flex-xy-center light-background">
                     <div class="card card-shadow">
                         <div class="header">
                             <div class="header-image-text-div">
                                     <img class="header-image" src="../Assets/images/dore.jpg" alt="" srcset="">
```
Comment (tanishq20) [inline Doc/card.html]: `alt` and `srcset` can't be empty.
(Apply this to all in rest code)

File: Doc/card.html [hunk 8]
```diff
@@ -219,6 +367,36 @@ <h4>by Fujiko Fujio</h4>
                         </div>
                     </div>
                 </div>
+                <pre>
+                    <code class="html">
+            <div class="component-display-div-row maple-flex-xy-center light-background">
+                <div class="card card-shadow">
+                    <div class="header">
+                        <div class="header-image-text-div">
+                                <img class="header-image" src="../Assets/images/dore.jpg">
```
Comment (tanishq20) [inline Doc/card.html]: `alt` missing
(Apply this to all in rest code)

**Review by tanishq20 — COMMENTED** [review]
Everything looks good. Suggested very minor changes.

**Review by tanishq20 — APPROVED** [review]

**Review by shubhamkumar648 — APPROVED** [review]
as i saw already modify your changes! no need to change required , code looks  good and readable , you can merge

**Review by sadathshariff — APPROVED** [review]
Looks good

**Pull request merged**